CHANGELOG.md

```diff
@@ -4,6 +4,30 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.8] - 2022-10-20
+### Added
+
+- [Web Crypto API] can now be used on Node.js if avalible [#284]
```

Comment:
The new lines after subsection are redundant and not consistent with entries for other versions.

Reply:
This, and the other documentation, leave it ambiguous about which API will be used when both are detected. The big table in lib.rs's module doc is actively misleading, if I am understanding correctly that the webcrypto API is preferred if detected.

Reply:
That's fair. I've fixed the spacing and spelling issues. I've also moved the WebCrypto stuff to the `Changed` section and actually elaborated on what we changed.

Let me know if it looks good to y'all.

I've also opened #295 to update the table in the Rust Docs.